src/widgets/settingspages/HighlightingPage.cpp

```diff
@@ -90,7 +107,7 @@ HighlightingPage::HighlightingPage()
                     "Play notification sounds and highlight messages from "
                     "certain users.\n"
                     "User highlights are prioritized over message "
-                    "highlights.");
+                    "and badge highlights.");
```

Comment:
What's the exact order here? This message says that User higlights are the most important, but then there's no information whether message or badge highlights are more important.

Reply:
I have updated this, please re-review [`46295e9` (#1704)](https://github.com/Chatterino/chatterino2/pull/1704/commits/46295e9e6827739a87ac18180d6853adb6e6a0a8)

Reply:
That explains it well, thank you for your time :+1: